### ISSUE TITLE
"Object doesn’t support this action" in IE10

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#249]
![CleanShot 2020-02-14 at 16 35 16@2x](https://user-images.githubusercontent.com/1079135/74544780-034f7500-4f48-11ea-803d-3a9574992d32.jpg)

Only when using push state.

This somehow fails:

```js
var his = window.history;
var pushState = "pushState";
var dis = window.dispatchEvent;
var addEventListenerFunc = window.addEventListener;

var stateListener = function(type) {
  var orig = his[type];
  return function() {
    var rv = orig.apply(this, arguments);
    var eventttt = new Event(type);
    eventttt.arguments = arguments;
    dis(eventttt);
    return rv;
  };
};

his.pushState = stateListener(pushState);

addEventListenerFunc(
  pushState,
  function() {
    console.log("pushState");
  },
  false
);

window.history.pushState({ "page_id": 2 }, "Page 2", "/page/2");
```

**Comment by adriaanvanrossum** [comment]
I think I found a solution: https://stackoverflow.com/questions/26596123/internet-explorer-9-10-11-event-constructor-doesnt-work

```js
function createNewEvent(eventName) {
    var event;
    if (typeof(Event) === 'function') {
        event = new Event(eventName);
    } else {
        event = document.createEvent('Event');
        event.initEvent(eventName, true, true);
    }
    return event;
}
```

**Comment by adriaanvanrossum** [comment]
Fixed in 575b03543bbd3959a2f94a9a5f6bcb348060cd33
